deployments/bsctestnet/VenusLens.json

```diff
@@ -0,0 +1,1015 @@
+{
+  "address": "0x11c8dC3DcA87E8205ec01e6d79Be9442d31701d3",
```

Comment:
Since this is a new deployment we should also bump the package version

Reply:
yeah bumped it